fakedata.cabal

```diff
@@ -609,6 +610,7 @@ library
     , template-haskell
     , text
     , time
+    , transformers
```

Comment:
Can you do the changes in `package.yaml` instead since we generate the cabal file from there ?

Reply:
Should I commit the updated cabal file as well?

Reply:
Yes, the generated cabal file needs to be updated.

Reply:
Also, can you add a performance note about this (similar to what you mentioned in the PR description):

```
Using this monad transformer, we were able to share a single FakerSettings, including the cache, across lots of Fake values, without having to rewrite everything in terms of IO. This was a massive performance improvement. Since it was useful to us, I thought it would be good to contribute back to upstream.
```

Basically,  give an example of a good performance and a non good performance code.

Reply:
Alright, I've moved those changes to `package.yaml`, including a couple of dependencies that were only in the `.cabal` file.

Reply:
I've now added a README section which describes how to use `FakeT` for better performance - let me know what you think.